sources/s63/sections/01-introduction.adoc

```diff
@@ -74,27 +78,26 @@ The Scheme Administrator (SA), of which there can only be one, authenticates the
 image::image-01.png[]
 
 
+[bibliography]
 === References
 
-// TODO: convert
-
-[1] S57 edition 3.1: IHO Transfer Standard for Digital Hydrographic Data, International Hydrographic Organization (www.iho.int)
+* [[[iho_ts_dhd,(1)IHO S57]]], S57 edition 3.1: IHO Transfer Standard for Digital Hydrographic Data, International Hydrographic Organization (www.iho.int)
 
-[2] Digital Signature Standard (DSS), FIPS Pub 186 (www.itl.nist.gov/div897/pubs/fip186.htm)
+* [[[dss,(2)FIPS Pub 186]]], Digital Signature Standard (DSS), FIPS Pub 186 (www.itl.nist.gov/div897/pubs/fip186.htm)
```

Comment:
Instead of `FIPS Pub 186` you have to write `NIST FIPS 186` for auto-fetching to work.